common/global_module_defs.hpp

```diff
@@ -0,0 +1,18 @@
+/*
+This is the file where default resolution values are defined
+*/
+
+
+//setting default width
+static constexpr int DEFAULT_WIDTH=2560;
+
+//setting default height
+static constexpr int DEFAULT_HEIGHT=1440;
```

Comment:
What are these for?

Reply:
@mhuzai Actually I was using it before for files like gldemo/plugin.cpp coz it was giving error like there must be same datatype static and constexpr to use it in the file. So , i put it there and it worked. Now I added #DEFINE so i guess it should do the task so I will remove it. :)